lib/authn/util/poll.ts

```diff
@@ -82,6 +83,48 @@ export function getPollFn(sdk, res: AuthnTransactionState, ref) {
       });
     }
 
+    const delayNextPoll = (ms) => {
+      // no need for extra logic in non-iOS environments, just continue polling
+      if (!isIOS()) {
+        return delayFn(ms);
+      }
+
+      let timeoutId: NodeJS.Timeout;
```

Comment:
Does `ReturnType<typeof setTimeout>` work better? This way TS sees the correct type at compile-time no matter which environment (Node/browser) we target.